scripts/gtcadis.py

```diff
@@ -0,0 +1,212 @@
+             'nb93': 'nb93', 'mo92': 'mo92', 'mo94': 'mo94',
+             'mo95': 'mo95', 'mo96': 'mo96', 'mo97': 'mo97',
+             'mo98': 'mo98', 'mo100': 'mo100', 'snnat': 'sn',
+             'ta181': 'ta181', 'w182': 'w182', 'w183': 'w183',
+             'w184': 'w184', 'w186': 'w186', 'au197': 'au197',
+             'pb206': 'pb206', 'pb207': 'pb207', 'pb208': 'pb208',
+             'bi209': 'bi209'}
+
+    names_dict = {nucname.id(value): key for key, value in names.iteritems()}
+    return names_dict
+
+
+def _cards(source):
+    cards = {"block1": {"isn": 16,
+                        "maxscm": '3E8',
+                        "maxlcm": '6E8',
+                        },
+             "block3": {"lib": "xsf21",  # name of cross section library
+                        "lng": 175,
+                        "maxord": 5,
+                        "ihm": 227,
+                        "iht": 10,
+                        "ihs": 11,
+                        "ifido": 1,
+                        "ititl": 1,
+                        "i2lp1": 0,
+                        "savbxs": 0,
+                        "kwikrd": 0
+                        },
+             "block5": {"source": source,
+                        "ith": 1,
+                        "isct": 5}
+             }
+    return cards
+
+
+def step1(cfg):
+
+    # Get user-input from config file
+    geom = cfg['step1']['geom_file']
+    cells = [cfg['step1']['src_cell']]
+    src_vol = [float(cfg['step1']['src_vol'])]
+    xmesh = cfg['step1']['xmesh'].split(',')
+    ymesh = cfg['step1']['ymesh'].split(',')
+    zmesh = cfg['step1']['zmesh'].split(',')
+
+    # Create structured mesh
+    sc = [np.linspace(float(xmesh[0]), float(xmesh[1]), float(xmesh[2]) + 1),
+          np.linspace(float(ymesh[0]), float(ymesh[1]), float(ymesh[2]) + 1),
+          np.linspace(float(zmesh[0]), float(zmesh[1]), float(zmesh[2]) + 1)]
+    m = Mesh(structured=True, structured_coords=sc)
+    m.mesh.save("blank_mesh.h5m")
+
+    # Generate 42 photon energy bins [eV]
+    #  First bin has been replaced with 1 for log interpolation
+
+    photon_bins = np.array([1e-6, 0.01, 0.02, 0.03, 0.045, 0.06, 0.07, 0.075, 0.1, 0.15,
+                            0.2, 0.3, 0.4, 0.45, 0.51, 0.512, 0.6, 0.7, 0.8, 1, 1.33, 1.34,
+                            1.5, 1.66, 2, 2.5, 3, 3.5, 4, 4.5, 5, 5.5, 6, 6.5, 7, 7.5, 8, 10,
+                            12, 14, 20, 30, 50])
+    # ICRP 74 flux-to-dose conversion factors
+    de = np.array([0.01, 0.015, 0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.1, 0.15, 0.2, 0.3,
+                   0.4, 0.5, 0.6, 0.8, 1, 2, 4, 6, 8, 10])
+    df = np.array([0.0485, 0.1254, 0.205, 0.2999, 0.3381, 0.3572, 0.378, 0.4066, 0.4399, 0.5172,
+                   0.7523, 1.0041, 1.5083, 1.9958, 2.4657, 2.9082, 3.7269, 4.4834, 7.4896,
+                   12.0153, 15.9873, 19.9191, 23.76])
+    # Convert to Sv/s per photon FLUX (not fluence)
+    flux_magnitude = 1E12
```

Comment:
This changed from 1e-12 to 1e12 in the most recent commit - which is correct?

Related: what is the purpose of this anyway?